scripts/run-dev.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# ./scripts/run-dev.sh quickly setup cluster for ACK development. It
+# ensures that a K8s cluster is accessible, then configures the ACK CRD, RBAC
+# but not install the controller
+
+set -Eeo pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$SCRIPTS_DIR/.."
+
+AWS_SERVICE=$(echo "${AWS_SERVICE:-""}" | tr '[:upper:]' '[:lower:]')
+
+CONTROLLER_NAMESPACE=${CONTROLLER_NAMESPACE:-"ack-system"}
+
+CLUSTER_NAME="ack-dev-cluster-$AWS_SERVICE"
+
+source "$SCRIPTS_DIR/lib/aws.sh"
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/config.sh"
+source "$SCRIPTS_DIR/lib/logging.sh"
+
+source "$SCRIPTS_DIR/controller-setup.sh"
+source "$SCRIPTS_DIR/kind.sh"
+
+ensure_cluster() {
+      info_msg "Creating KIND cluster ..."
+      setup_kind_cluster "$CLUSTER_NAME" "$CONTROLLER_NAMESPACE"
+
+      info_msg "Installing CRDs , common and RBAC manifest..."
+      install_crd_and_rbac "$CONTROLLER_NAMESPACE"
+}
+
+run() {
+    ensure_aws_credentials
+    ensure_cluster
+    local kubeconfig_path="$ROOT_DIR/build/clusters/$CLUSTER_NAME/kubeconfig"
+    info_msg "Before running the controller, you need kubeconfig and aws credentials."
+    info_msg "After executing the above source command to set the kubeconfig to environment, "
+    info_msg "run the following command to start the controller from the controller repo:"
+    echo ""
+    echo "go run ./cmd/controller/main.go --aws-region eu-central-1 --log-level debug --enable-development-logging"
```

Comment:
nit: Instead of hardcoding `eu-central-1`, get the region from the config YAML ([like this](https://github.com/aws-controllers-k8s/test-infra/blob/main/scripts/run-e2e-tests.sh#L58))

Reply:
fixed [rename to create-dev-cluster.sh](https://github.com/aws-controllers-k8s/test-infra/pull/338/commits/177a044a9a14e15081b3b637eada300916cced39)